src/main/java/com/projetonetflix/projetonetflix/model/entity/History.java

```diff
@@ -0,0 +1,65 @@
+package com.projetonetflix.projetonetflix.model.entity;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "HISTORY")
+public class History {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "IDT_HISTORY")
+    private Long id;
+
+    @Column(name = "DAT_HISTORY")
+    private Date datHistory;
```

Comment:
Aqui não é errado usar Date, mas a API nova do java (8+) LocalDate e LocalDateTime tem algumas funções que facilitam a manipulação e o parse. Como vc vai ser do time, é bom vc ir se habituando com ela pois é o nosso padrão.

Algumas comparações:
https://stackoverflow.com/questions/28730136/should-i-use-java-util-date-or-switch-to-java-time-localdate

Reply:
Beleza vou usar ela e dar uma olhada nela Go!